plugins/inputs/tail/tail.go

```diff
@@ -197,20 +234,9 @@ func (t *Tail) tailNewFiles(fromBeginning bool) error {
 				continue
 			}
 
-			var seek *tail.SeekInfo
-			if !t.Pipe && !fromBeginning {
```

Comment:
Has the test for `t.Pipe` been intentionally removed? After the last update it seems to try seeking from pipes which leads to `illegal seek` errors.

Reply:
It looks like this was not intentionally. Please create bug report and if possible also a PR to fix this.

Reply:
@fetzerch could you please open a new issue so we don't loose track of this!?